cirq-core/cirq/vis/heatmap.py

```diff
@@ -33,6 +35,7 @@
 import numpy as np
 from mpl_toolkits import axes_grid1
 
+import cirq
```

Comment:
You may need to put this under `if TYPE_CHECKING:` to avoid lint error on unused cirq.
See for example https://github.com/quantumlib/Cirq/blob/d7f6d7d78825d32645108f37a182a469a752b686/cirq-core/cirq/circuits/circuit.py#L67-L68

Reply:
Yes, thanks. But since we have removed `_as_set`, this import itself is not needed. So I've removed it.